zenoh-ts/src/config.ts

```diff
@@ -0,0 +1,44 @@
+//
+// Copyright (c) 2024 ZettaScale Technology
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+//
+// Contributors:
+//   ZettaScale Zenoh Team, <zenoh@zettascale.tech>
+//
+
+//  ██████  ██████  ███    ██ ███████ ██  ██████
+// ██      ██    ██ ████   ██ ██      ██ ██
+// ██      ██    ██ ██ ██  ██ █████   ██ ██   ███
+// ██      ██    ██ ██  ██ ██ ██      ██ ██    ██
+//  ██████  ██████  ██   ████ ██      ██  ██████
+
+export class Config {
+  /**
+   * The configuration for a Zenoh Session.
+   */
+  locator: string;
+
+  /**
+   * @hidden
```

Comment:
I don't think `@hidden` is a valid tag in JSDoc: https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html.

VS Code somehow interprets it as an author tag and links to this GitHub profile: https://github.com/hidden.

Reply:
I was following TypeDoc, 
but I will change it to @ignore as thats covered by both `JsDoc` and `TypeDoc`
https://typedoc.org/tags/hidden/#see-also

Reply:
Implemented in https://github.com/eclipse-zenoh/zenoh-ts/pull/5